### PR TITLE
Port Grisell player (K31R) from Fortran

### PR DESCRIPTION
**Pull request by RobPDev**
This is player K31R (Grisel), ported from Fortran at PyCon UK 2017

### CODE REVIEW

File: axelrod/strategies/axelrod_second.py [hunk 1]
```diff
@@ -98,6 +98,40 @@ def strategy(opponent: Player) -> Action:
         return random_choice(1 - defection_prop)
 
 
+class Grisell(Player):
+    """
+    Submitted to Axelrod's second tournament by Gail Grisell.  It came 23rd was written in 10 lines of BASIC.
```
Comment (drvinceknight) [inline axelrod/strategies/axelrod_second.py]: `and was written`

Also can you return at 80 characters please? (same for rest of docstring)

**Comment by drvinceknight** [comment]
> Unless I'm mistaken this strategy is already implemented under the HardGoByMajority strategy types:

Apologies: it's the `GoByMajority` class in that same file.

I've opened https://github.com/Axelrod-Python/axelrod-fortran/pull/67 on the fortran library that checks this (deterministic strategies are tested).

**Comment by RobPDev** [comment]
I think I've made the necessary changes, if I've missed something then let me know

**Comment by RobPDev** [comment]
Hopefully that's everything, but if you need me to reorder it or if I made a mistake with the reference then I'm happy to correct it.

**Comment by drvinceknight** [comment]
That's awesome. Thanks again! Going through the Fortran and understanding what strategies they are is super helpful. :+1:

File: docs/reference/overview_of_strategies.rst [hunk 2]
```diff
@@ -101,7 +101,7 @@ repository.
    :header: "Original Code", "Author", "Axelrod Library Name"
 
    "GRASR_", "Unknown", "Not Implemented"
-   "K31R_", "Gail Grisell", "Not Implemented"
+   "K31R_", "Gail Grisell", ":class:`HardGoByMajority <axelrod.strategies.gobymajority.HardGoByMajority>`"
```
Comment (drvinceknight) [inline docs/reference/overview_of_strategies.rst]: Apologies: this should actually be `GoByMajority`

Comment (RobPDev) [reply docs/reference/overview_of_strategies.rst]: Ah, I missed that when I made the changes in 708ee0bc0d961d66d0267081356f5ca198a60431.  I'll change it now.

**Comment by meatballs** [comment]
Looks good! Thanks @RobPDev

**Comment by drvinceknight** [comment]
Thanks @RobPDev :+1:

**Pull request merged**